### ISSUE TITLE
Browse mode: reading wrong next focusable element if the previous focusable element is changed from display block to none

### ISSUE THREAD

**Issue by shliepic** [nvaccess/nvda#10343]
### Steps to reproduce: Firefox + NVDA
[Firefox+NVDA weired reading while InfoBubble followed by a focusable element (open in Firefox).txt](https://github.com/nvaccess/nvda/files/3694877/Firefox%2BNVDA.weired.reading.while.InfoBubble.followed.by.a.focusable.element.open.in.Firefox.txt)
1. Open the attach file in Firefox.
2. Tabbing through the page until a info bubble shows. The info bubble contains a long text that  NVDA will split the text into three part while in browse mode (arrow down to read the text).
![image](https://user-images.githubusercontent.com/56236027/66276333-84d71e00-e857-11e9-8df0-b89a4b479d79.png)
3. Keep arrow down until the infobubble fade.
4. At this time the focus should be on "Dan Jian" focusable span element. However, NVDA read the "Designate one or more..." focusable element instead.
![image](https://user-images.githubusercontent.com/56236027/66276372-062eb080-e858-11e9-953e-b197ed59ba89.png)

The interested thing is this issue won't be able to reproduce on jsfiddler and codepen, which we suspect it's because of the iframe. The issue won't occur if the infobubble is rendered within an anchor element.

#### Have you tried any other versions of NVDA? If so, please report their behaviors. 
Tried the older version of NVDA 2018.3.2. The issue doesn't occur.

**Comment by lukaszgo1** [comment]
cc @jcsteh

**Comment by Adriani90** [comment]
cc: @MarcoZehe

**Comment by MarcoZehe** [comment]
Confirmed, however, this only affects speech. Braille is immediately focused on the correct span element.
Reproduced with NVDA 2019.3 alpha and Firefox 71 Nightly, so this is nothing that is adversely affected by our big changes in Firefox 70.
